build/v1/types.go

```diff
@@ -899,14 +899,27 @@ type BuildConfigStatus struct {
 	LastVersion int64 `json:"lastVersion" protobuf:"varint,1,opt,name=lastVersion"`
 }
 
+// SecretLocalReference contains information that points to the local secret being used
+type SecretLocalReference struct {
+	// Name is the name of the resource being referenced
```

Comment:
"name of the secret in the same namespace being reference"

Reply:
fixed.